scripts/test/run-integration-tests.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -e
+OS_OVERRIDE=$(go env GOOS)
+pushd ./test/integration-new
+
+echo "Running integration test with the following configuration:
+KUBE CONFIG: $KUBECONFIG
+CLUSTER_NAME: $CLUSTER_NAME
+AWS_REGION: $AWS_REGION
+OS_OVERRIDE: $OS_OVERRIDE"
+
+while getopts "f:" arg; do
```

Comment:
nit: we can get this variable from ENV variable as well like the other ones.

Reply:
Hi, so I planned on passing it as a flag instead of ENV variable as I thought it would be more dynamic in comparison to other ENV variables that would more likely remain constant for multiple runs of the script. The focus parameter would most likely change for multiple runs, making passing the flag a simpler alternative. Let me know if you think otherwise or think that we should keep everything consistent as ENV.